java/ql/src/semmle/code/java/Statement.qll

```diff
@@ -704,10 +706,10 @@ class ExprStmt extends Stmt, @exprstmt {
   /** Gets the expression of this expression statement. */
   Expr getExpr() { result.getParent() = this }
 
-  /** Gets a printable representation of this statement. May include more detail than `toString()`. */
   override string pp() { result = "...;" }
 
-  /** This statement's Halstead ID (used to compute Halstead metrics). */
+  override string toString() { result = "...;" }
```

Comment:
It isn't at all clear that this is describing an expression statement -- I think simply "expression statement" would be better.

Reply:
I went for `<Expr>;` as suggested below.